hack/download-charts.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e -u
+
+: "${MAISTRA_RELEASE_STREAM:=$1}"
+: "${MAISTRA_REPO:=$2}"
+: "${MAISTRA_COMMIT:=$3}"
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+REPO_ROOT=$(dirname "${SCRIPT_DIR}")
+HELM_DIR="${REPO_ROOT}/resources/charts/${MAISTRA_RELEASE_STREAM}"
+
+ISTIO_FILE="${MAISTRA_COMMIT}.zip"
+ISTIO_URL="${MAISTRA_REPO}/archive/${MAISTRA_COMMIT}.zip"
+WORK_DIR=$(mktemp -d)
+EXTRACT_DIR="${MAISTRA_REPO##*/}-${MAISTRA_COMMIT}"
+EXTRACT_CMD="unzip -q ${ISTIO_FILE} ${EXTRACT_DIR}/manifests/charts/* ${EXTRACT_DIR}/manifests/addons/dashboards/*"
+
+function downloadIstioCharts() {
+  if [ -d "${HELM_DIR}" ] ; then
```

Comment:
superflous. You can `rm -rf` directly without checking.

Reply:
fixed